content/services/project-consulting.mdx

```diff
@@ -117,7 +117,7 @@ links:
   <ProjectEstimationSection/>
 </ContentSection>
 
-<ContentSection title="Getting Started">
+<ContentSection title="Contact Us">
```

Comment:
Can we drop the "Weekly office hours" in this section? I feel like it's a little out of place for the project consulting page. It's not like we have people always on deck to help with collaboration questions. It would be better for them to email us first and we can schedule meeting during office hours if we think it's appropriate.